cpu/atmega_common/periph/pwm.c

```diff
@@ -58,6 +60,38 @@ static inline unsigned get_prescaler(pwm_t dev, uint32_t *scale)
     return pre;
 }
 
+static inline void compute_cra_and_crb(pwm_t dev, uint8_t pre)
+{
+    uint8_t cra = _BV(WGM1) | _BV(WGM0);
+    uint8_t crb = pre;
+
+    if (pwm_conf[dev].pin_ch[0] != GPIO_UNDEF) {
+        cra |= _BV(COMA1);
```

Comment:
using `cra |= _BV(COMA1) | _BV(COMA0)` would result in `PMW_RIGHT`. Dropping the anti-flicker work around for 0% and 100% PWM duty makes implementing `PWM_RIGHT` (as a potential follow up PR) much easier.

Reply:
(Similar for the other PWM channel, just replace `A` with `B`.)